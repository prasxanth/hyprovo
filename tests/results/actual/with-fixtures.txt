2021-11-24 14:41:21.418 - INFO - Pass ✓... (test-mul-variable): mul pass
2021-11-24 14:41:21.420 - ERROR - Fail ✗... (test-mul-variable): mul fail
2021-11-24 14:41:21.425 - INFO - Pass ✓... (test-div-variable): div pass
2021-11-24 14:41:21.427 - ERROR - Fail ✗... (test-div-variable): div fail
2021-11-24 14:41:21.431 - ERROR - Fail ✗... (test-mul-variable): mul pass
2021-11-24 14:41:21.433 - ERROR - Fail ✗... (test-mul-variable): mul fail
2021-11-24 14:41:21.437 - ERROR - Fail ✗... (test-div-variable): div pass
2021-11-24 14:41:21.440 - ERROR - Fail ✗... (test-div-variable): div fail
