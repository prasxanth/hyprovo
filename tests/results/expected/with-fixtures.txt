2021-11-24 14:10:30.900 - INFO - Pass ✓... (test-mul-variable): mul pass
2021-11-24 14:10:30.903 - ERROR - Fail ✗... (test-mul-variable): mul fail
2021-11-24 14:10:30.907 - INFO - Pass ✓... (test-div-variable): div pass
2021-11-24 14:10:30.909 - ERROR - Fail ✗... (test-div-variable): div fail
2021-11-24 14:10:30.916 - ERROR - Fail ✗... (test-mul-variable): mul pass
2021-11-24 14:10:30.918 - ERROR - Fail ✗... (test-mul-variable): mul fail
2021-11-24 14:10:30.922 - ERROR - Fail ✗... (test-div-variable): div pass
2021-11-24 14:10:30.924 - ERROR - Fail ✗... (test-div-variable): div fail
